browse/templates/base.html

```diff
@@ -184,6 +184,51 @@ <h2 id="mobilemenulabel">quick links</h2>
       </div>
     </footer>
   </div>
+
+  <script type="text/javascript">
+
+    function setCookie(cname, cvalue, exdays) {
```

Comment:
Can you do this with [local storage ](https://developer.mozilla.org/en-US/docs/Web/API/Window/localStorage) instead of a cookie?

Reply:
Also, can you move this to base so that all our projects can use this?